### ISSUE TITLE
[mpileup] no input file/data given in Estimating False Negative Rate

### ISSUE THREAD

**Issue by AlejandroMS17** [adamjorr/zypy#46]
Hi Adam,

I am trying to run the "bcftools mpileup" step to estimate the false negative rate, but I am getting this error

`amorales@dacelo:/disks/dacelo/data/03_analyses_RB7_7_fa_as_reference_chapter_2_thesis/06_false_negative_rate_estimation/test_false_negative_rate$ bcftools mpileup -a 'AD,DP' -T RB7_7_fa_fai_sites_to_mutate.bed -q3 -Q13 -Ou -f RB7_7.fa | bcftools call -m -A -p 0 -P 0.015 -O z -o RB7_7_bcftools.vcf.gz
Note: none of --samples-file, --ploidy or --ploidy-file given, assuming all sites are diploid
[mpileup] no input file/data given
Failed to open -: unknown file type
amorales@dacelo:/disks/dacelo/data/03_analyses_RB7_7_fa_as_reference_chapter_2_thesis/06_false_negative_rate_estimation/test_false_negative_rate$`

Did you get this error too?

Do we need to provide a BAM file?
`Usage: bcftools mpileup [options] in1.bam [in2.bam [...]]`

Thank you!.

**Comment by adamjorr** [comment]
Yes, you should provide the bam file as input there. Thanks for reporting this, I'll fix it in the readme ASAP

**Comment by AlejandroMS17** [comment]
Great!, thanks for the prompt reply.

**Comment by AlejandroMS17** [comment]
One more question in regards of the BAM file as input file. Does the BAM file needs to contains all the samples?. 

Because in the following step I got the following error:

`amorales@dacelo:/disks/dacelo/data/03_analyses_RB7_7_fa_as_reference_chapter_2_thesis/06_false_negative_rate_estimation/test_false_negative_rate$ bcftools query -s GL/M -f '%CHROM\t%POS\t%REF\t%FIRST_ALT\t[%TGT]\n' RB7_7_bcftools.vcf.gz | tr '/' '\t' > RB7_7_fa_fai_sites_to_mutate.txt
Sample name mismatch: sample #1 not found in the header`

The input BAM file that I used for this test, did not contain all the samples. 

I am guessing I need to provide that information in:

`bcftools query -s GL/M`

**Comment by adamjorr** [comment]
Yes, for that step you should pass the sample name of the root sample with the `-s` option.
